Python-packages/covidcast-py/covidcast/covidcast.py

```diff
@@ -7,12 +7,22 @@
 import pandas as pd
 import numpy as np
 from delphi_epidata import Epidata
+from delphi_epidata.delphi_epidata import _HEADERS
+from pkg_resources import get_distribution, DistributionNotFound
 from epiweeks import Week
 
 from .errors import NoDataWarning
 
+
 # Point API requests to the default endpoint
-Epidata.BASE_URL = "https://api.covidcast.cmu.edu/epidata"
+Epidata.BASE_URL = "https://api.covidcast.cmu.edu/epidata/api.php"
```

Comment:
```suggestion
Epidata.BASE_URL = "https://api.covidcast.cmu.edu/epidata"
```
whoops, i mustve edited the wrong version of the file which got `api.php` back into the `BASE_URL`